backend/dynamic-ui/http-data-modelling.ftd

```diff
@@ -0,0 +1,255 @@
+-- import: fifthtry.github.io/admonitions as cbox
+
+-- ds.page: Basics of `http` processor and `data modelling`
+
+In this part, we will delve into essential concepts like the `http` processor
+and `data modelling` through a concise yet illuminating example.
+
+To facilitate your learning journey, we will take a pratical scenario involving 
+an array of records. Each record has two key attributes: the **country-name** 
+and it's corresponding **capital**, in the form of JSON data.
+
+-- ftd.text: Click to see the JSON data
+link: https://famous-loincloth-ox.cyclic.app/
+open-in-new-tab: true
```

Comment:
Why are you torturing reader? 

Use `ds.code` and show the JSON right here.

Reply:
Sure, I have displayed it.

```json
-- ds.code:
lang: json
copy: false

[
  {
    "name": "India",
    "capital": "New Delhi"
  },
  {
    "name": "Sri Lanka",
    "capital": "Colombo"
  },
  {
    "name": "Nepal",
    "capital": "Kathmandu"
  },
  {
    "name": "Bangladesh",
    "capital": "Dhaka"
  },
  {
    "name": "Indonesia",
    "capital": "Jakarta"
  },
  {
    "name": "Maldives",
    "capital": "Mali"
  }
]
```